### PR TITLE
1149 meditrak data time

### PR DESCRIPTION
**Pull request by edmofro**
### Issue #: 
https://github.com/beyondessential/tupaia-backlog/issues/1149, subtasks 10, 11, 13, 14, 18

### CODE REVIEW

File: packages/database/src/migrations/20200218025613-AddDataGroupsToDataSource.js
Comment (edmofro) [inline packages/database/src/migrations/20200218025613-AddDataGroupsToDataSource.js:28]: This is just so if anyone uses it for reference, they're aware to also ignore DateOfData

File: packages/meditrak-server/src/routes/importSurveyResponses/importSurveyResponses.js
Comment (edmofro) [inline packages/meditrak-server/src/routes/importSurveyResponses/importSurveyResponses.js:189]: Just whitespace changes - @EMcQ-BES this is where to test you have prettier running

**Review by EMcQ-BES — APPROVED** [review]

**Comment by edmofro** [comment]
Will hold off merging until your base branch is approved @EMcQ-BES

**Comment by edmofro** [comment]
Note to self: I also need to get tests passing before this can be merged

**Comment by edmofro** [comment]
Tests should now be passing

**Comment by EMcQ-BES** [comment]
Merging this so I can use some of the utils you've written

**Pull request merged**
